trunk/problems/MAXWELL/GALERKIN/makefile

```diff
@@ -4,27 +4,33 @@
 
 COMPLEX        ?= 1
 DEBUG_MODE     ?= 0
+HP3D_USE_OPENMP    ?= 0
+HP3D_USE_INTEL_MKL ?= 0
 
 ifeq ($(COMPLEX),0)
 $(error COMPLEX FLAG MUST BE 1 FOR MAXWELL APPLICATION)
 endif
 
+ifeq ($(COMPILER),INTEL)
+ifeq ($(DEBUG),1)
+FFLAGS += -gen-interface -warn none -warn interfaces
```

Comment:
I also add `-check bounds` to the debugging flags

Reply:
We can add that, yes. I think we should probably move all of these additional FFLAGS into m_options files and then read them from there only.